[9.0.2] build fails while unknown symbol

/builddir/build/BUILD/gvm-libs-9.0.2/nasl/nasl_crypto2.c: In function 'nasl_aes128_gcm_encrypt':
/builddir/build/BUILD/gvm-libs-9.0.2/nasl/nasl_crypto2.c:1572:51: error: 'GCRY_CIPHER_MODE_GCM' undeclared (first use in this function)
   return encrypt_data (lexic, GCRY_CIPHER_AES128, GCRY_CIPHER_MODE_GCM);
                                                   ^
/builddir/build/BUILD/gvm-libs-9.0.2/nasl/nasl_crypto2.c:1572:51: note: each undeclared identifier is reported only once for each function it appears in
/builddir/build/BUILD/gvm-libs-9.0.2/nasl/nasl_crypto2.c: In function 'nasl_aes256_gcm_encrypt':
/builddir/build/BUILD/gvm-libs-9.0.2/nasl/nasl_crypto2.c:1578:51: error: 'GCRY_CIPHER_MODE_GCM' undeclared (first use in this function)
   return encrypt_data (lexic, GCRY_CIPHER_AES256, GCRY_CIPHER_MODE_GCM);
                                                   ^
/builddir/build/BUILD/gvm-libs-9.0.2/nasl/nasl_crypto2.c: In function 'nasl_aes128_gcm_encrypt':
/builddir/build/BUILD/gvm-libs-9.0.2/nasl/nasl_crypto2.c:1573:1: error: control reaches end of non-void function [-Werror=return-type]
 }
 ^
/builddir/build/BUILD/gvm-libs-9.0.2/nasl/nasl_crypto2.c: In function 'nasl_aes256_gcm_encrypt':
/builddir/build/BUILD/gvm-libs-9.0.2/nasl/nasl_crypto2.c:1579:1: error: control reaches end of non-void function [-Werror=return-type]
 }
 ^
cc1: all warnings being treated as errors
make[2]: *** [nasl/CMakeFiles/openvas_nasl_shared.dir/nasl_crypto2.c.o] Error 1

For the full build log, see the attachments. 

[build.log](https://github.com/greenbone/gvm-libs/files/1796698/build.log)
[hw_info.log](https://github.com/greenbone/gvm-libs/files/1796699/hw_info.log)
[installed_pkgs.log](https://github.com/greenbone/gvm-libs/files/1796700/installed_pkgs.log)
[root.log](https://github.com/greenbone/gvm-libs/files/1796701/root.log)
[state.log](https://github.com/greenbone/gvm-libs/files/1796702/state.log)

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -258,6 +258,13 @@ if (NOT MINGW)
     execute_process (COMMAND libgcrypt-config --cflags
       OUTPUT_VARIABLE GCRYPT_CFLAGS
       OUTPUT_STRIP_TRAILING_WHITESPACE)
+    execute_process (COMMAND libgcrypt-config --version
+      OUTPUT_VARIABLE GCRYPT_VERSION
+      OUTPUT_STRIP_TRAILING_WHITESPACE)
+    message (STATUS "  found libgcrypt, version ${GCRYPT_VERSION}")
+    if (GCRYPT_VERSION VERSION_LESS "1.6")
+      message (SEND_ERROR "libgcrypt 1.6 or greater is required")
+    endif (GCRYPT_VERSION VERSION_LESS "1.6")
   endif (NOT GCRYPT)
 
 endif (NOT MINGW)

--- a/INSTALL
+++ b/INSTALL
@@ -22,7 +22,7 @@ General build environment:
 Specific development libraries:
 * libglib >= 2.32
 * libgnutls >= 3.2.15
-* libgcrypt
+* libgcrypt >= 1.6
 * zlib
 * libpcap
 * libgpgme >= 1.1.2

